meinberlin/apps/ideas/templates/meinberlin_ideas/idea_moderate_form.html

```diff
@@ -27,8 +27,8 @@ <h1>{% translate 'Feedback on this idea' %}</h1>
                 {{ form.media }}
             {% endfor %}
 
-            {% include 'meinberlin_contrib/includes/form_field.html' with field=forms.moderateable.moderator_feedback %}
-            {% include 'meinberlin_contrib/includes/form_field.html' with field=forms.statement.statement %}
+            {% include 'meinberlin_contrib/includes/form_field.html' with field=forms.moderateable.moderator_status %}
+            {% include 'meinberlin_contrib/includes/form_field.html' with field=forms.moderator_feedback_text.feedback_text %}
```

Comment:
The `moderator_feedback_text` is the form name, so it should be `feedback_text`.